packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -15,6 +17,7 @@ exports.sourceNodes = async (
     headers,
     params,
     concurrentFileRequests,
+    preview,
```

Comment:
I'm just wondering if we do need to make this feature opt-in? If we figure out a way to not have to spawn extra server - we could just always attach route handler

Reply:
I think if we're able to use the existing express instance there's no need to opt in.